content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,180 @@
+
+Once an open source creation has progressed through the three phases above, the virtuous cycle, driven by commercial dependency and reinvestment, is completed. At this point, the project has reached a steady-state of robust community development, bug fixes and security updates are optimized through the many eyeballs on a project, lucrative partnerships can be formed, and much more. 
+
+Commercial dependency results in users, users buy solutions that generate profit, and profits increase resources to invest back in the community. A healthy community can also encourage partnerships and cross-pollination between projects that improve quality.
+
+&nbsp;
+
+# Ingredients of open source ecosystem success
+
+
+Creating unique code that solves a problem is job number one in pursuing the optimal ecosystem for an open source creation. For example, OpenStack has helped countless organizations centralize resources in the cloud, resulting in cost reductions for in-house storage and compute resources. Likewise, NFV technology is helping telecommunications companies reduce their reliance on costly proprietary components in their technology stacks. 
+
+Once the problem-solving creation exists, focus can be placed on building community and contributions, instilling confidence for commercial adoption, and adherence to open source culture:
+
+## Building community/ growing contributions 
+
+To build a healthy community around an open source project, developer training and recruitment are key, as are [defining guidelines for inbound contributions](https://opensource.guide/how-to-contribute/), and disciplined development practices such as version management, build and test automation, documentation, on-ramps for contribution, and tracking issues and revisions.
+
+"You need to give people ways to get involved with your projects that don't require them to have a Ph.D. or to have been working in a similar area for 25 years. You need ways for them to get involved quickly. That means that you need really good setup documentation, and it also means having active and healthy forums and responsive maintainers." Ian Varley, Software Architect, SalesForce.
+
+The [Contributor Covenant](http://contributor-covenant.org/) is a rock solid code of conduct and contributor guidelines document that is used by [over 40,000 open source projects](http://contributor-covenant.org/adopters/), including Kubernetes, Rails, and Swift. Likewise, organizations such as TODO Group at The Linux Foundation and Black Duck have extensive experience with setting inbound contribution guidelines. 
```

Comment:
Isn't Black Duck a product? How can it have experience with setting contribution guidelines?

Reply:
I think this comes from this article: https://www.linuxfoundation.org/resources/open-source-guides/setting-an-open-source-strategy/

"There are many external resources that can help you flesh out your open source strategy, and the good news is that many of them are free. For example, Black Duck consults with thousands of businesses on setting open source strategies and offers guides and tutorials on the topic" However, the links don't point to a live URL (might be offering those guidelines at some point, but not anymore). I will delete black duck from this sentence. Thanks!